packages/esm-commons-lib/src/components/banner-tags/.patient-status-tag.test.tsx

```diff
@@ -81,69 +44,7 @@ describe('PatientStatusBannerTag', () => {
       isError: false,
     });
 
-    (usePatientOutcome as jest.Mock).mockReturnValue({
-      patientOutcome: 'HIV negative infant discharged from PMTCT',
-    });
-
-    (usePatientFamilyNames as jest.Mock).mockReturnValue({
-      childrenNames: [],
-      motherName: null,
-      patientAge: null,
-      patientGender: null,
-      isLoading: false,
-      isError: false,
-    });
-
     render(<PatientStatusBannerTag patientUuid={hivPositiveSampleUuid} />);
     expect(screen.getByText('HIV Negative')).toBeInTheDocument();
```

Comment:
Thanks @lucyjemutai , you can also add in other props and see if the values will be validated